### PR TITLE
PowerShell: Create language file for PowerShell

### PR DESCRIPTION
**Pull request by pareksha**
This creates a basic language file for PowerShell in
coala language definitions.

Closes https://github.com/coala/coala/issues/5582

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/bearlib/languages/definitions/PowerShell.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class PowerShell:
+    extensions = '.ps1',
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {'<#': '#>'}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = {'"': '"', "'": "'"}
```
Comment (jayvdb) [inline coalib/bearlib/languages/definitions/PowerShell.py]: can you show me some info about these being multiline ?

Comment (pareksha) [reply coalib/bearlib/languages/definitions/PowerShell.py]: Sure, I found some info at https://www.sconstantinou.com/powershell-quotes/

Comment (jayvdb) [reply coalib/bearlib/languages/definitions/PowerShell.py]: Great.
Then you can do `multiline_string_delimiters = string_delimiters` to make it more clear they are identical.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5586/commits/02aba47f65adea2b0e591ade75be4f65c6319890

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**